src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int TEST_SIZE = 6;
+
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[TEST_SIZE];
+        figures[TEST_SIZE - 1] = figureSupplier.getDefaultFigure();
+        for (int i = 0; i < TEST_SIZE - 1; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
+        }
```

Comment:
There's no need to draw the figure immediately after creation. Consider separating the creation and drawing of figures into separate loops for better separation of concerns.